<?xml version="1.0"?>
<robot name="box">

  <contact name="box_surface">
    <link name="world"/>
    <point>0.600 0.400 0.096   0.600 -0.400 0.096   1.12 0.400 0.096   1.12 -0.400 0.096 </point>
    <shape> 4 1 3 2 0   </shape>
  </contact>
</robot>
